src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -16699,6 +16702,13 @@ void Unit::SetControlled(bool apply, UnitState state)
             default:
                 break;
         }
+
+        if (GetTypeId() == TYPEID_PLAYER)
+        {
+            uint32 pinginthismoment = fabs(ToPlayer()->GetLastMoveClientTimestamp() - ToPlayer()->GetLastMoveServerTimestamp()) / 1000000;
+            ToPlayer()->SetRootACKUpd(pinginthismoment);
+            //TC_LOG_INFO("anticheat", "Latency = %u", pinginthismoment);
```

Comment:
should we convert this log for AC and enable it?

Reply:
i think not needed, just detele it too